src/Spline.cpp

```diff
@@ -200,10 +200,25 @@ double base::geometry::SplineBase::getCurveLength(double relative_resolution) co
     return length;
 }
 
-double SplineBase::getUnitParameter()
```

Comment:
This removal brakes planning/corridor_planner.

I didn't get what is the alternative here.

@D-Alex 

>/home/planthaber/dfki/rock/entern/planning/corridor_planner/src/main.cc: In function 'void markCurve(base::geometry::Spline<3>&, int, std::vector<RGBColor>&, RGBColor)':
/home/planthaber/dfki/rock/entern/planning/corridor_planner/src/main.cc:51:27: error: 'class base::geometry::Spline<3>' has no member named 'getUnitParameter'
/home/planthaber/dfki/rock/entern/planning/corridor_planner/src/main.cc: In function 'void outputPlan(int, int, const string&, const std::vector<unsigned char>&, corridor_planner::Plan&)':
/home/planthaber/dfki/rock/entern/planning/corridor_planner/src/main.cc:165:43: error: 'class base::geometry::Spline<3>' has no member named 'getUnitParameter'
/home/planthaber/dfki/rock/entern/planning/corridor_planner/src/main.cc:182:47: error: 'class base::geometry::Spline<3>' has no member named 'getUnitParameter'
ICECC[7551] 10:21:09: Compiled on 192.168.128.14
src/CMakeFiles/plan_corridors.dir/build.make:54: recipe for target 'src/CMakeFiles/plan_corridors.dir/main.cc.o' failed